aries-site/src/examples/templates/dashboards/components/UpdateNotificationsList.js

```diff
@@ -0,0 +1,27 @@
+import PropTypes from 'prop-types';
+import { Box, List } from 'grommet';
+import { UpdateItem } from './UpdateItem';
+
+export const UpdateNotificationsList = ({
```

Comment:
Not for this PR, but I think we can also name this to make this more generic and reusable.

Reply:
agree I wanted to first get this written out then go back and clean up my brain seems to work better that way

Reply:
Yep. I agree. Just planting seeds.